bench/wtperf/misc.c

```diff
@@ -54,7 +54,7 @@ setup_log_file(CONFIG *cfg)
 		return (ret);
 
 	/* Use line buffering for the log file. */
-	(void)setvbuf(cfg->logf, NULL, _IOLBF, 32);
+	(void)setvbuf(cfg->logf, NULL, _IOLBF, 1024);
```

Comment:
According to the MSDN doc @agorrod noted in the ticket, only the sizes of 0 were a problem.  But having them all the same buffer size is goodl